docs/_src/tutorials/tutorials/6.md

```diff
@@ -249,10 +236,10 @@ print_answers(prediction, details="minimum")
 
 This [Haystack](https://github.com/deepset-ai/haystack/) notebook was made with love by [deepset](https://deepset.ai/) in Berlin, Germany
 
-We bring NLP to the industry via open source!  
-Our focus: Industry specific language models & large scale QA systems.  
-  
-Some of our other work: 
+We bring NLP to the industry via open source!
```

Comment:
I think removing the tab here causes the next line `Our focus:` to be put on the same line as `We bring ...`, which I don't think is intended. Could you add a space between the lines or add back the tab?

Reply:
I added a space between the lines. The removal was from `trailing-whitespace` pre-commit hook. So couldn't put back the tab without disabling it.